autoload/crystalline.vim

```diff
@@ -461,6 +525,13 @@ function! crystalline#apply_current_theme() abort
   catch /^Vim\%((\a\+)\)\=:E118/
     " theme does not use autoload function
   endtry
+
+
+  for [l:sep_hi_group, l:groups_ab] in items(g:crystalline_sep_hi_groups)
+    call crystalline#generate_sep_hi(l:groups_ab[0], l:groups_ab[1])
```

Comment:
You shouldn't have to regenerate separator highlight groups here since they'll be regenerated by `generate_sep_hi` now that `g:crystalline_sep_hi_groups` is empty, the way it is now I believe they'll be regenerated twice.

Reply:
You're right, of course.